db.json

```diff
@@ -147,6 +147,20 @@
         }
       ],
       "standards": [
+        {
+          "name": "Cookie Store API",
+          "href": "https://wicg.github.io/cookie-store/",
```

Comment:
It'll need to be a `spec.whatwg.org` URL. I suggest `https://cookiestore.spec.whatwg.org/`.